shell.nix

```diff
@@ -73,5 +76,20 @@ pkgs.mkShell rec {
     echo $(which $CXX)
     echo -e "\nIncludeOS package:"
     echo ${includeos}
+    echo -e "\n---------------------- Network privileges  ---------------------"
+    echo "The vmrunner for IncludeOS tests requires bridged networking for full functionality."
+    echo "The following commands requiring sudo privileges can be used to set this up:"
+    echo "1. the qemu-bridge-helper needs sudo to create a bridge. Can be enabled with:"
+    echo "   sudo chmod u+s ${pkgs.qemu}/libexec/qemu-bridge-helper"
+    echo "2. bridge43 must exist. Can be set up with \$create_bridge :"
+    echo "   ${vmrunner.create_bridge}"
+    echo "3. /etc/qemu/bridge.conf must contain this line:"
+    echo "   allow bridge43"
+    echo ""
+    echo "Some tests require ping, which requires premissions to send raw packets. On some hosts"
+    echo "this is not enabled by default for iputils provided by nix. It can be enabled with:"
+    echo "4. sudo setcap cap_net_raw+ep ${pkgs.iputils}/bin/ping"
+    echo " "
+    echo
```

Comment:
Perhaps ./test.sh should have a `--no-net` parameter. If you don't have these things set up you'll get expected failures mixed with other test failures.

Reply:
Yes, good point. What I would prefer even more though, is to have a smaller test suite work with userspace networking. Many of the tests will work with that, but not the IPv6 ones, and it will require some rewriting of those tests and possibly vmrunner changes, so that's a bit of work. I added a task for that here: https://github.com/includeos/IncludeOS/issues/2267